src/ctrls/PlaybookMagicCtrl.js

```diff
@@ -43,43 +43,67 @@ class PlaybookMagicCtrl extends Controller {
-        console.log("**************************************************")
-        githubUrl = "https://github.com/Domnom/nodegit-tester";
+            // -- Prompt for the app github URL if it has not been provided
+            if (!githubUrl || !ValidationService.isGithubUrl(githubUrl))
+            {
+                let githubUrlAnswer = await inquirer.prompt({
+                    type : "input",
+                    name : "githubUrl",
+                    default: "https://github.com/Domnom/nodegit-tester",
+                    message: "What github repo would you like to convert?",
+                    validate: (val)=>{
+                        return ValidationService.isGithubUrl(val) ? true: 'Please enter a valid github URL';
+                    },
+                })
 
-        if (!githubUrl || !ValidationService.isGithubUrl(githubUrl))
-        {
-            // -- Prompt user for a github repo
-            // githubUrl = inquirer.prompt("YO WHERE THE GITHUB URL AT")
-            let githubUrlAnswer = await inquirer.prompt({
-                type : "input",
-                name : "githubUrl",
-                default: "https://github.com/Domnom/nodegit-tester",
-                message: "What github repo would you like to convert?",
-                validate: (val)=>{
-                    return ValidationService.isGithubUrl(val) ? true: 'Please enter a valid github URL';
-                },
+                githubUrl = githubUrlAnswer.githubUrl;
+            }
+
+            if (!blueprintGithubUrl || !ValidationService.isGithubUrl(githubUrl))
+            {
+                let blueprintGithubUrlAnswer = await inquirer.prompt({
+                    type : "input",
+                    name : "blueprintGithubUrl",
+                    default: "https://github.com/Domnom/nodegit-tester-blueprint",
+                    message: "What github repo would you like to save the blueprints to?",
+                    validate: (val)=>{
+                        return ValidationService.isGithubUrl(val) ? true: 'Please enter a valid github URL';
+                    },
+                })
+
+                blueprintGithubUrl = blueprintGithubUrlAnswer.blueprintGithubUrl;
+            }
 
-            })
+            /**
+             * @step 2. Create or clone a git repo to store/update the blueprints 
+             */
+            const blueprintRepoData = await NodeGitService.createOrCloneBlueprintRepoFromGithubUrl((blueprintGithubUrl ? blueprintGithubUrl : githubUrl), !!blueprintGithubUrl);
+            
+            
+            /**
+             * @step 3. Create the blueprints folder with the github URL
+             */
+            const blueprintRepoBranchUrl = await NodeGitService.createBlueprintsFolderFromGithubUrl(githubUrl, blueprintRepoData);
 
-            githubUrl = githubUrlAnswer.githubUrl;
+
+            process.stderr.write("\n============ Processing complete ============\n\n")
```

Comment:
You can use chalk inside console log for printing colors if you like 
E.g `chalk.green`